lib/vrl/compiler/src/expression/function_call.rs

```diff
@@ -530,16 +530,16 @@ impl DiagnosticError for Error {
 
                 let mut notes = vec![];
 
-                let call = format!("{}{}({}))", function_ident, abort, args);
+                let call = format!("{}{}({})", function_ident, abort, args);
 
                 notes.append(&mut Note::solution(
-                    "guard against invalid type at runtime",
+                    "ensuring an appropriate type at runtime",
                     vec![format!("{} = {}", argument, guard), call.clone()],
                 ));
 
                 if let Some(coerce) = coerce {
                     notes.append(&mut Note::solution(
-                        "coerce with default value",
+                        "using a default value as a fallback",
```

Comment:
The example we give here is: `to_string(.foo) ?? "default"`, so it's not only using a default value, but also coercing.